app/react/src/server/cra-config.js

```diff
@@ -62,7 +62,14 @@ export const getRules = extensions => rules =>
   }, []);
 
 const getStyleRules = getRules(cssExtensions.concat(cssModuleExtensions));
-const getTypeScriptRules = getRules(typeScriptExtensions);
+const getTypeScriptRules = typescriptRules => {
+  const rules = getRules(typeScriptExtensions)(typescriptRules);
+  // Add support for using TypeScript in the `.storybook` folder.
+  const babelRule = rules[0];
+  if (babelRule.include !== 'string') return rules;
+  babelRule.include = [babelRule.include, path.join(babelRule.include, '../.storybook')];
```

Comment:
We need to use the `configDir` prop coming from the preset in order to put the reall config dir that user could provide via the cli. It should be already a resolved path.

Reply:
@igor-dv I've added this in the same way that the `react-native` app manages it, but using `commander`. Please let me know what you think.